src/components/BookForm.jsx

```diff
@@ -76,11 +75,11 @@ const BookForm = () => {
             <option>Economy</option>
           </select>
         </div>
-        <div className="col-md-2 pb-5 pb-md-0">
-          <button type="submit" onClick={addBookToList}>
-            ADD BOOK
-          </button>
-        </div>
+
+        <button type="submit" className="add-book-btn btn" onClick={addBookToList}>
+          ADD BOOK
+        </button>
+
```

Comment:
- [ ] Kindly be noticed that adding a book is not working please fix that.
- [ ] Also the `ADD BOOK` button goes over the categories select menu please fix that.

![Screenshot from 2022-07-15 00-09-29](https://user-images.githubusercontent.com/33752735/179097016-2a5e532f-dab7-4931-b70c-3f6d70c57f0f.png)